cmd/agent/main.go

```diff
@@ -19,15 +19,21 @@ func main() {
 		preferDriver         string
 		allocAllDevices      bool
 		devicepluginPreStart bool
+		simpleMode           bool
```

Comment:
Replacing the `simpleMode` parameter with `localERIDiscovery` can more accurately express the functionality of managing only on-node resources.
Then the `exposeERIs` can rename to `exposedLocalERIs` or `localERIPatterns` if support regex configuration.